Added jsonResponse field and JSON validation schema (#11)
testproject-io/addons: restful-api-client/README.md

	Parameter is *false* by default.
	If set to `true`, RESTful client will accept self signed and untrusted SSL certificate presented by the server when sending a request.
	
* `schemaValidationOutputFilePath` - This parameter is the file path for the result of the JSON schema validation.
    
    If empty, it will not create a file for the validation output.
    
* `schemaPath` - This parameter is the path for the JSON Schema file.
       
    This parameter can have either a FULL path for a local schema file, or a URL directly to the schema file.
    If a URL is provided, it will download the Schema locally to a temp file before doing the validation.

>     The schema validation will work on the JSON response provided in the JsonResponse field.

Reviewer: ```suggestion
    This parameter is for a *full** path of a local schema file, or a URL pointing to a remote schema file.
    If a URL is provided, it will download the schema locally to a temporary file, before performing the validation.
    Schema validation is performed on the JSON response object provided in the `JsonResponse` field.
```

> * `createFile` - This parameter is for creating a file for the validation output or not.

Reviewer: ```suggestion
* `createFile` - This parameter is a boolean flag to indicate that a validation results output is required.
```

>     If set to `true`, and the `schemaValidationOutput` parameter is provided, a file holding the validation result will be present
>     in the given location.

Reviewer: ```suggestion
    If set to `true`, and the `schemaValidationOutput` parameter is set, a file holding the validation result will be created
    in the provided location.
```

        

#### Output Fields

* `response` - The full response from the server or the extracted node/value found using expression specified in `jsonPath` parameter.

    > If `jsonPath` is set, `response` will be limited to the node/value found using the provided expression.\
    If the value specified is not found or the the response or the response is not a valid JSON, this field will be empty.
      

> * `jsonResponse` - The full response from the server or the extracted node/value found using expression specified in `jsonPath` parameter as an JSON object.
> 
>     > This will hold either the respone body, or the node/property of the jsonPath as an JSON Object.

Reviewer: ```suggestion
* `jsonResponse` - The full response from the server or the extracted node/value found using expression specified in `jsonPath` parameter as a JSON object.
    > This parameter holds the whole response body, or the node/property requested via jsonPath as a JSON object.
```

* Server response status (taken from the `status` field)
* Response body or “No body/value was returned by the server” when it's absent.

> * The result of violations from the json Schema, if provided, or if the validation found none.

Reviewer: ```suggestion
* Report of violations discovered using provided json schema, or a statement that none were found.
```
